src/wallet/interface.ts

```diff
@@ -85,6 +87,12 @@ export interface SendTransactionParams extends WalletBaseParams {
    * @default Native token transfer transaction
    */
   data?: string;
+
+  /**
+   * Gas limit multiplier
+   * Applies only for native tokens swap
+   */
+  gasMultiplier?: Amount;
```

Comment:
describe wallet params abstracted from swap

\+ provide `@default` behavior info

```suggestion
  /**
   * Gas limit multiplier
   *
   * @default ...
   */
  gasMultiplier?: Amount;
```